docs/book/toc.md

```diff
@@ -64,6 +64,7 @@
 
 * [📜 Component guide](stacks-and-components/component-guide/component-guide.md)
   * [Integration overview](stacks-and-components/component-guide/integration-overview.md)
+  * [Implementing custom components](stacks-and-components/component-guide/writing-custom-components.md)
```

Comment:
rename the file as well, then?

Reply:
Ah this was the old removed page actually